src/coreclr/nativeaot/System.Private.CoreLib/src/System/RuntimeType.cs

```diff
@@ -46,6 +46,7 @@ internal void Free()
             RuntimeImports.RhHandleFree(_runtimeTypeInfoHandle);
         }
 
+        [FeatureSwitchDefinition("System.Reflection")]
         private static bool IsReflectionDisabled => false;
```

Comment:
Doesn't this property reflect the negated value of that feature switch? Does this account for that somehow?

```xml
  <type fullname="System.RuntimeType" feature="System.Reflection" featurevalue="false">
    <method signature="System.Boolean get_IsReflectionDisabled()" body="stub" value="true" />
  </type>
```

Reply:
> No, great catch, that one can't be handled by FeatureSwitchDefinition. I'm surprised this didn't cause test failures in ci. @MichalStrehovsky

We don't test things with reflection disabled since that mode is unsupported. So if this enabled reflection unconditionally, I'm not surprised it wasn't caught. I'd be surprised if it was the other way and nothing failed.